### PR TITLE
Support for volume API

### PR DESCRIPTION
**Pull request by marcuslinke**
fixes issue #329

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/command/CreateVolumeCmd.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+     * @param driverOpts
+     *            - A mapping of driver options and values. These options are passed directly to the driver and are
+     *            driver specific.
+     */
+    public CreateVolumeCmd withDriverOpts(Map<String, String> driverOpts);
+
+    public static interface Exec extends DockerCmdSyncExec<CreateVolumeCmd, CreateVolumeResponse> {
+    }
+
+}
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/CreateVolumeCmd.java]: new line

**Comment by KostyaSha** [comment]
@marcuslinke have you successfully run build locally with static analysing profile enabled? :)

**Comment by marcuslinke** [comment]
@KostyaSha Sorry, what do you mean. Doesn't a simple `mvn clean install` should be sufficient?

**Comment by KostyaSha** [comment]
@marcuslinke install will pollute your local repo :) `mvn -Pdocker-java-analysis package` or just `mvn package` should be enough. But probably checkstyle doesn't contain new line check. In general having new lines in the end of file is good tone (starting from time when NL was expected as end of stream/resource) and excludes red icon.

**Comment by marcuslinke** [comment]
OK. newlines fixed now.

**Comment by KostyaSha** [comment]
@marcuslinke could you merge #409 and rebase this PR? :)

File: src/main/java/com/github/dockerjava/api/command/CreateVolumeCmd.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package com.github.dockerjava.api.command;
+
+import java.util.Map;
+
+import javax.annotation.CheckForNull;
+
+public interface CreateVolumeCmd extends SyncDockerCmd<CreateVolumeResponse> {
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/CreateVolumeCmd.java]: Maybe add javadoc when apparead? 1.21?

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/CreateVolumeCmd.java]: Can do myself later.

**Comment by KostyaSha** [comment]
Is travis building merged state? let's check.

**Comment by KostyaSha** [comment]
Sorry, merge conflict. Hoped that it wouldn't.

**Comment by KostyaSha** [comment]
Added NL in missed file, fixuped two commits in one, merged manually to master. 
This PR is really merged.

**Pull request closed without merging**